src/dotnet-structuring.library/Helpers/Logging/Logging.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace dotnet_structuring.library.Helpers.Logging
+{
+    public class Logging
+    {
+        public delegate void StructuringHandler(object sender, string log);
```

Comment:
don't name the delegate `StructuringHandler` like the class ´StructuringHandler`

Reply:
There is no class such as `StructuringHandler`